.ci.yaml

```diff
@@ -109,28 +109,6 @@ platform_properties:
       os: Mac-12.0
       device_os: iOS-15.1
       xcode: 13a233
-  mac_ios32:
```

Comment:
Do we run these tests on release branches?

Reply:
Yes. We can let this roll through the release process, or we can CP it into the next releases